docs/data/opendata.md

```diff
@@ -0,0 +1,19 @@
+---
+---
+layout: datasource
+tablename: example_table_name
+title: Example Table
+---
+<!--No need to put a header; the title in the front matter (above) will be used as a header-->
+
+A number of data sources come from the [opendata.dc.gov](http://opendata.dc.gov/) project. The OpenData project describes itself as a site where:
+
+> ...the District of Columbia government shares hundreds of datasets. DC realizes that data’s greatest value comes from having it freely shared among agencies, federal and regional governments and with the public to the extent possible when considering safety, privacy and security. 
+
+Users are invited to analyze the available datasets and build apps using OpenData's APIs.
+
+We have used the following data sources from OpenData. Please click the links below to read about each source in further detail.
+
+[Building Permits]({{site.baseurl}}/building_permits)
```

Comment:
Almost there. `{{site.baseurl}}` matches with the 'docs' folder when this is deployed. `docs` becomes `housinginsights.org`. So to get these urls to work correctly, you need to add the /data folder to the path i.e. `{{site.baseurl}}/data/building_permits`